clients/java/signalr/src/main/java/Logger.java

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public interface Logger {
+    void log(LogLevel logLevel, String message);
```

Comment:
Can we add an overload to this that takes format specifiers that we flow through `String.format`?

```java
void log(LogLevel logLevel, String format, Object... args);
```

That means we can make interpolated log strings a little cleaner:

```java
logger.log(LogLevel.Trace, "Some message (some value = %d)", someValue);
```